gallery/visualtests/testUtils.js

```diff
@@ -177,6 +181,56 @@ module.exports = {
       await page.mouse.move(0, 0);
     }
 
+    async function fillFile(path, numBytes) {
+      const MAX_BUFFER_SIZE = 1 << 20, // 1 MiB
+          writeStream = fs.createWriteStream(path);
+
+      let buffer;
+
+      for (let i = 0; i < numBytes; i += MAX_BUFFER_SIZE) {
+        const bufferSize = Math.min(MAX_BUFFER_SIZE, numBytes - i);
+
+        if (!(buffer && buffer.length === bufferSize)) {
+          buffer = await Buffer.alloc(bufferSize);
+        }
+
+        writeStream.write(buffer);
+      }
+
+      return new Promise((resolve, reject) => {
+        writeStream.on('finish', resolve);
+        writeStream.on('error', reject);
+
+        writeStream.end();
+      });
+    }
+
+    let files, tmpDir;
+
+    async function buildFiles() {
```

Comment:
The name of this is a little bit vague.  How about `buildUploadableFiles`?

Reply:
> The name of this is a little bit vague. How about `buildUploadableFiles`?

Sounds good, renamed here: [3aa5bf0](https://github.com/sonatype/sonatype-react-shared-components/pull/850/commits/3aa5bf06c3cf1c31626e3847fdfbb16f06c41393)